LLama/Model/IModelCache.cs

```diff
@@ -86,15 +87,10 @@ public interface IModelCache : IDisposable
 
     /// <summary>
     /// Attempt to get a model that's expected to be loaded
+    /// The callers assumes responsiblilty for the lifetime of the model at this point if it exists in the cache
     /// </summary>
     /// <param name="modeId"></param>
     /// <param name="model"></param>
     /// <returns></returns>

```

Comment:
```suggestion
    /// <summary>
    /// Attempt to get a model that's has already been loaded. The caller assumes responsibility for disposing the returned model.
    /// </summary>
    /// <param name="modeId">The modelId which was passed into <see cref="LoadModel"/></param>
    /// <param name="model">The model, if it was already loaded, otherwise null</param>
    /// <returns>true, if the model was available in the cache. Otherwise false.</returns>
```